wurst/_items/MageMasher.wurst

```diff
@@ -0,0 +1,58 @@
+package MageMasher
+
+import AbilityObjEditing
+import ObjectIdGenerator
+import ClosureEvents
+import DamageType
+import ID
+import IttDummyCaster
+import Orders
+import StandardTextTags
+import LinkedList
+
+let MANABURN_ID = compiletime(ABIL_ID_GEN.next())
+let DRAIN_AMOUNT = 6.
+
+LinkedList<int> MASHER_TARGET_CLASSES = asList(
+                            UNIT_MAGE,
```

Comment:
Do not hard code lists like this. It's not resilient to more skins. Use `instanceof` instead.